### PR TITLE
Bugs #13768: package Pastis standalone with java 17

### PR DESCRIPTION
**Pull request by ebernard**

### CODE REVIEW

**Review by bbenaissa — APPROVED** [review]

File: api/api-pastis/pastis-standalone/pom.xml [hunk 1]
```diff
@@ -272,8 +269,8 @@
                                 <configuration>
                                     <target>
                                         <get
-                                                src="https://download.java.net/openjdk/jdk11/ri/openjdk-11+28_windows-x64_bin.zip"
-                                                dest="${project.build.directory}/jdk11.zip"
+                                                src="https://download.oracle.com/java/17/archive/jdk-17.0.12_windows-x64_bin.zip"
```
Comment (mohatizaoui) [inline api/api-pastis/pastis-standalone/pom.xml]: pourquoi ce n'est pas openJDK 17 ?
https://download.java.net/openjdk/jdk17/ri/openjdk-17+35_windows-x64_bin.zip

Comment (mohatizaoui) [reply api/api-pastis/pastis-standalone/pom.xml:272]: https://download.java.net/openjdk/jdk17/ri/openjdk-17+35_windows-x64_bin.zip

Comment (marob) [reply api/api-pastis/pastis-standalone/pom.xml]: Ou sinon la version Temurin https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.13%2B11/OpenJDK17U-jdk_x64_windows_hotspot_17.0.13_11.zip ?

**Review by marob — APPROVED** [review]

File: api/api-pastis/pastis-standalone/pom.xml (hunk 1 above)
Comment (ebernard) [reply api/api-pastis/pastis-standalone/pom.xml]: J'ai mis Temurin.

**Pull request merged**